config-paths.js

```diff
@@ -0,0 +1,23 @@
+const { app } = require('electron');
+const { homedir } = require('os');
+const { resolve } = require('path');
+const { existsSync } = require('fs');
+
+const home = homedir();
+const root = resolve(app.getPath('appData'), 'HyperTerm');
+const legacyConfig = resolve(home, './.hyperterm.js');
```

Comment:
Good catch. We should be using joins on the rest of the paths as well, since XDG paths should be absolute as well, correct?